src/Samples/SubModel.Core/Program.fs

```diff
@@ -42,6 +98,35 @@ module Counter =
   ]
 
 
+type [<AllowNullLiteral>] public CounterViewModel(initialModel, dispatch) as this =
+  inherit ViewModelBase<Counter.Model, Counter.Msg>(initialModel, dispatch)
+
+  new() = CounterViewModel(Counter.init, ignore)
+
+  member _.StepSize
+    with get() = this.getValue (fun m -> m.StepSize)
+    and set(v) = this.setValue ((fun v _m -> Counter.Msg.SetStepSize v), v)
+  member _.CounterValue = this.getValue (fun m -> m.Count)
+  member _.Increment = this.cmd((fun _ _ -> Counter.Increment |> ValueSome), (fun _ _ -> true))
+  member _.Decrement = this.cmd((fun _ _ -> Counter.Decrement |> ValueSome), (fun _ _ -> true))
+  member _.Reset = this.cmd((fun _ _ -> Counter.Reset |> ValueSome), (fun _ -> Counter.canReset))
```

Comment:
This is an example of a class with get, set, and cmd. I think I can shorten set's because we always have the value available directly in the callback.